spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -801,6 +807,12 @@ Users are allowed to reuse existing JAX-RS endpoints for participant method defi
 In this case the LRA implementation MUST ensure that invoking these methods outside of the 
 implementation of the LRA specification will not influence any running LRA.
 
+Specifically, developers should NEVER call any JAX-RS endpoint for participant callback methods
+(@Compensate, @Complete, @Status, @Forget, and @AfterLRA) where they
+add a header defined by the Java constant `LRA_HTTP_RECOVERY_HEADER`.
```

Comment:
for AfterLRA it is LRA_HTTP_ENDED_CONTEXT_HEADER, but I don't think it's important to mention it here. Just noting.